docs/pywbemcli/commands.rst

```diff
@@ -1781,36 +1793,56 @@ The ``statistics server-show`` command displays the current WBEM server gathered
 
-    $ pywbemcli -s http://localhost
-
-    pywbemcli> statistics server-show
-
-    WBEM server CIM Operation Statistical Data
-    +-------------+--------------+--------------+---------------+---------------+------------------------+
-    |   Operation |       Server |     Provider |       Request |      Response | Operation Name         |
-    |       Count |   time(usec) |   Time(usec) |   size(bytes) |   size(bytes) |                        |
-    |-------------+--------------+--------------+---------------+---------------+------------------------|
-    |          34 |        459.2 |          0   |         379.5 |       14505.3 | EnumerateClassNames    |
-    |          45 |       7719.4 |       3264.8 |         381.3 |        3739.4 | EnumerateInstanceNames |
-    |          62 |       2241.3 |         73.2 |         384.9 |        2449.1 | EnumerateInstances     |
-    |           4 |        229.8 |          0   |         582   |        4339   | GetClass               |
-    |           8 |        471.4 |         38.1 |         862   |        3128   | GetInstance            |
-    |           8 |        884.9 |         25.6 |        3523   |         376.4 | ModifyInstance         |
-    |           5 |       7372.6 |          0   |         595.2 |        3192   | OpenEnumerateInstances |
-    +-------------+--------------+--------------+---------------+---------------+------------------------+
+    $ pywbemcli -s http://localhost statistics server-show
+
+    Server statistics
+    Operation                 Count    Server Time    Provider Time    Request Size    Response Size
+                                              [ms]             [ms]             [B]              [B]
+    ----------------------  -------  -------------  ---------------  --------------  ---------------
+    GetClass                    315          0.239            0                 399             4124
+    GetInstance                   4          0.495            0.07              862             3128
+    ModifyInstance                4          0.795            0.024            3523              376
+    EnumerateInstances          172          1.459            0.341             383            10111
+    EnumerateInstanceNames      132          0.913            0.657             377             4102
+    OpenEnumerateInstances      156          1.986            0                 616            14506
+
+The **Operation** column shows the name of the CIM-XML operation.
+
+The **Count** column shows the number of operations executed.
+
+All time and size values in this report are average values across the number
+of operations executed, rounded to the precision shown.
+
+The **Server Time** column shows the total elapsed time the operation took from
+a perspective of the WBEM server. It has the same meaning as the Server Time
+value shown in the client statistics. This time includes time spent in the
+CIM object manager code and time spent in any providers.
```

Comment:
After reading the description of CIMOMElapsedTime in CIM_CIMOMStatisticalData.mof, it turns out it says that it does not include the provider time.
Can you please double check what Pegasus implemented for it?
**DISCUSSION**

Reply:
Will check

Reply:
**CONCLUSION:** Yes, OpenPegasus does treat the time in the CIM Object Manager coorectly by subtracting the provider time if there is any before saving the server time.  It is nod in Common/CIMMessage.cpp method endServer()

Reply:
Ok, great! That means we can go by the description of CIMOMElapsedTime for all CIMOM brands, which the current code does. RESOLVED.